source/connection-monitoring-and-pooling/tests/pool-clear-close-in-use.yml

```diff
@@ -0,0 +1,42 @@
+version: 1
+style: unit
+description: Pool clear MUST support lazily closing in use connections
+poolOptions:
+  backgroundThreadIntervalMS: 100
+operations:
+  - name: ready
+  - name: checkOut
+  - name: checkOut
+    label: conn
+  - name: clear
+    closeInUseConnections: true
+  - name: waitForEvent
+    event: ConnectionClosed
+    count: 2
+    timeout: 1000
+  - name: close
+events:
+  - type: ConnectionCheckedOut
+    connectionId: 1
+    address: 42
+  - type: ConnectionCheckedOut
+    connectionId: 2
+    address: 42
+  - type: ConnectionPoolCleared
+    address: 42
+  - type: ConnectionClosed
+    connectionId: 1
```

Comment:
since we have no ordering requirements on which connections get closed first, I think we need to omit the `connectionId` here.

Reply:
done